README.md

```diff
@@ -8,10 +8,10 @@ Synopsys Detect scripts to invoke Detect.
 ## Where can I get the latest release? ##
 
 *Available from GitHub for Linux by running:*  
-bash <(curl -s -L https://detect.synopsys.com/detect7.sh)
+bash <(curl -s -L https://detect.synopsys.com/detect9.sh)
 
 *Available from GitHub for Windows by running:*  
-powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://detect.synopsys.com/detect7.ps1?$(Get-Random) | iex; detect"
+powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://detect.synopsys.com/detect9.ps1?$(Get-Random) | iex; detect"
```

Comment:
Interesting that this change is going from 7 to 9. Was it not updated when 8 was released?

Reply:
Yes someone might have missed it in Detect 8 release